packages/react-core/src/components/Popover/Popover.tsx

```diff
@@ -14,6 +15,7 @@ import { ReactElement } from 'react';
 import { FocusTrap } from '../../helpers';
 import { Popper, getOpacityTransition } from '../../helpers/Popper/Popper';
 import { getUniqueId } from '../../helpers/util';
+import { Instance as TippyInstance, Props as TippyProps } from 'tippy.js';
```

Comment:
Can we find a way to copy the `TippyInstance` and `TippyProps` definitions instead?